README.md

```diff
@@ -39,32 +78,56 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
 
-## Learn More
+### Learn More
 
 You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
 
 To learn React, check out the [React documentation](https://reactjs.org/).
 
-### Code Splitting
+#### Code Splitting
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
 
-### Analyzing the Bundle Size
+#### Analyzing the Bundle Size
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
 
-### Making a Progressive Web App
+#### Making a Progressive Web App
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
 
-### Advanced Configuration
+#### Advanced Configuration
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
 
-### Deployment
+#### Deployment
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
 
-### `npm run build` fails to minify
+#### Troubleshooting
+
+How do you address the case when the following script fails to minify?
+
+```bash
+npm run build
+```
+
+See [here](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify) for troubleshooting tips.
+
+## 📚 Usage
+
+You can utilize the source code in this repository for getting started with Arcana Network SDKs. Use it to build a stand alone basic React application that integrates with the latest Auth and Storage SDK.  You may plug in additional functionality and try out various features of the SDKs.
+
+See [Arcana Documentation](https://docs.beta.arcana.network/) for user guides and SDK Reference Guides.  
+
+To get started with Arcana SDKs quickly, you can also check out other [code samples and templates](https://docs.beta.arcana.network/docs/overview_cs) in the `Developer Resources` section of the documentation.
+
+## 💡 Support
+
+For any queries regarding this repository usage, contact [Arcana support team](mailto:support@arcana.network).
```

Comment:
For any queries regarding this repository usage, contact the  [Arcana Support Team](mailto:support@arcana.network).